### PR TITLE
[Frontend][Paddle] [PaddlePaddle Hackathon 4] add convert support for p_norm/roi_align/softmax_with_cross_entropy

### PR DESCRIPTION
**Pull request by AndPuQing**
Add support for converting the following Paddle OPs.

- [p_norm](https://www.paddlepaddle.org.cn/documentation/docs/en/2.2/api/paddle/norm_en.html#norm)
- [roi_align](https://www.paddlepaddle.org.cn/documentation/docs/en/2.5rc/api/paddle/vision/ops/roi_align_en.html)
- [softmax_with_cross_entropy](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/softmax_with_cross_entropy_en.html#softmax-with-cross-entropy)

cc @jiangjiajun

### CODE REVIEW

File: tests/python/frontend/paddlepaddle/test_forward.py [hunk 1]
```diff
@@ -2302,5 +2302,112 @@ def forward(self, x, y):
...
+@tvm.testing.uses_gpu
+def test_forward_affine_channel():
+    class AffineChannel(nn.Layer):
+        def __init__(self):
+            super(AffineChannel, self).__init__()
+
+        @paddle.jit.to_static
+        def forward(self, inputs, scale, bias):
+            return paddle.fluid.layers.affine_channel(inputs, scale, bias)
+
```
Comment (jiangjiajun) [inline tests/python/frontend/paddlepaddle/test_forward.py:2314]: `fluid.layers.xxx` is deprecated, and will be removed in next version, please use other api, or just remove this operator

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

**Review by jiangjiajun — APPROVED** [review]
Please modify the title to "[Frontend][Paddle] [PaddlePaddle Hackathon 4] add convert support for p_norm/roi_align/softmax_with_cross_entropy", since the `affine_channel` is removed.

**Comment by jiangjiajun** [comment]
@junrushao Could you help to merge this PR? I found that I'am still a `contributor` not a `reviewer`...

**Review by junrushao — APPROVED** [review]
